src/release/rollups.md

```diff
@@ -0,0 +1,60 @@
+# Rollup Procedure
+
+## Background
+
+The Rust project has a policy that every pull request must be tested after merge
+before it can be pushed to master. As PR volume increases this can scale poorly,
+especially given the long (>2hr) current CI duration for Rust.
+
+Enter rollups - low risk changes (often doc fixes or other non-functional
+changes) are marked with the `rollup` command to bors (`@bors r+ rollup` to
+approve a PR and mark as a rollup, `@bors rollup` to mark a previously approved
+PR, `@bors rollup-` to unmark as a rollup).  'Performing a Rollup' then means
+collecting these changes into one PR and merging them all at once.
```

Comment:
It's worth mentioning that `rollup` is equivalent to `rollup=always` and `rollup-` to `rollup=maybe`. Finally, `rollup=never` is introduced later here but we should mention that people can use this to mark PRs that are risky, big, or e.g. perf sensitive as `rollup=never` (we prefer they add a reasoning for doing so however).